code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -1,3 +1,17 @@
+/client/verb/xeno_health_alerts()
+	set name = "Toggle Xeno Health Alerts"
+	set desc = "Toggles alerts when a low HP xeno is injured."
+	set category = "Alien"
+
+	prefs.toggles_alerts ^= ALERTS_XENO_HEALTH
+	prefs.save_preferences()
+
+	if(prefs.toggles_alerts & ALERTS_XENO_HEALTH)
+		to_chat(src, "<span class='xenodanger'>You will no longer receive hive alerts about badly wounded xenos.</span>")
+	else
+		to_chat(src, "<span class='xenodanger'>You will now receive hive alerts about badly wounded xenos.</span>")
+
+
```

Comment:
Use the pref menu

Reply:
I think that's far less intuitive; probably better to keep it here as it is exclusively concerned with aliens